CHANGELOG.md

```diff
@@ -7,13 +7,37 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 **Note**: Since GDK for Unreal v0.10.0, the changelog is published in both English and Chinese. The Chinese version of each changelog is shown after its English version.<br>
 **注意**：自虚幻引擎开发套件 v0.10.0 版本起，其日志提供中英文两个版本。每个日志的中文版本都置于英文版本之后。
 
+
 ## [`x.y.z`] - Unreleased
+
+### Breaking changes:
```

Comment:
This file was very messed up, basically had to reconstruct a whole new `## [`x.y.z`] - Unreleased` from a previous `Unreleased` block that had 0.13.0 and post-0.13.0 changes interleaved.